blog/se1/index.qmd

```diff
@@ -0,0 +1,28 @@
+---
+author: Caleb Kendra, Alish Chhetri, and Evan Nelson
+title: Programming Over Time
+date: '2023-11-29'
+date-format: YYYY
+categories: [post, developer development, software engineering]
+description: <em>This is our first blog about the `Software Engineering at Google` book!</em>
+page-layout: full
+---
+
+# Summary
+
+In the preface of "Software Engineering at Google" in the Programming Over Time section we are first introduced to the idea that "Software Engineering" is not just the act of creating code, but also the creation of processes that allow us to create maintainable and sustainable software processes. In this sense, when we release a version of the software we created, we don't want that software to be the final version but rather a building block upon which we can improve. In order to succeed in this goal of sustainability and growth, the book emphasizes three points to be thinking of while creating software:
```

Comment:
- Please use a link instead of the name of the article in quotation marks.
- Please make sure that you use a "hard line-wrap" and not a "soft line-wrap". The way that you have written this paragraph makes it very hard to review and very hard to see who wrote what.
- What is the meaning of the phrase "creating code"? This seems too wordy.